README.md

```diff
@@ -9,11 +9,48 @@ The original project is available at [WineHQ](https://source.winehq.org/git/vkd3
 
 ## Priorities
 
-Performance and compatibility are important targets, at the expense of compatibility with older drivers and systems.
-Modern Vulkan extensions and features are aggressively made use of to improve performance and compatibility.
+Performance and game compatibility are important targets, at the expense of compatibility with older drivers and systems.
+Modern Vulkan extensions and features are aggressively made use of to improve game performance and compatibility.
 It is recommended to use the very latest drivers you can get your hands on for the best experience.
 Backwards compatibility with the vkd3d standalone API is not a goal of this project.
 
+## Drivers
+
+There are some hard requirements on drivers to be able to implement D3D12 in a reasonably performant way.
+
+- Vulkan 1.1
+- `VK_EXT_descriptor_indexing` with at least 1000000 UpdateAfterBind descriptors for all types except UniformBuffer.
+  Essentially all features in `VkPhysicalDeviceDescriptorIndexingFeatures` must be supported.
+- `VK_KHR_timeline_semaphore`
+
+Some notable extensions that **should** be supported for optimal or correct behavior.
+These extensions will likely become mandatory later.
+
+- `VK_EXT_robustness2`
+- `VK_KHR_buffer_device_address`
+
+### AMD (RADV / ACO)
+
+For AMD, RADV is the recommended driver and the one that sees most testing on AMD GPUs.
+The recommendation here is to use a driver built from Git.
```

Comment:
Anything known issues with 20.2?

Reply:
Death Stranding SSE flag bug I think?

Reply:
ye, 20.2.1 was 14th Oct, that fix was Oct 21st.

Reply:
Ok.